packages/web3-eth/CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Moved `rpc_methods` tests to `web3-rpc-methods` (#5441)
 -   [setimmediate](https://github.com/yuzujs/setImmediate) package to polyfill [setImmediate](https://nodejs.org/api/timers.html#setimmediatecallback-args) for browsers (#5450)
 -   Implemented the logic for `transactionBlockTimeout` (#5294)
+-   Use subscription at `rejectIfBlockTimeout` when the provider supports subscription. Implement this as an experimental feature (if `enableExperimentalFeatures` is `true`). (#5481)
```

Comment:
Should `enableExperimentalFeatures` be an object for extensibility in case we want to add more experimental features in the future?
like
```
enableExperimentalFeatures = { 
    subscribeSupscription: boolean,
    ... // next experimental feature
}
```

Reply:
That sounds like a good suggestion :+1: 
Done!